cilium/cmd/encrypt_status.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/common"
+	"github.com/cilium/cilium/pkg/option"
+	"github.com/prometheus/procfs"
+	"github.com/vishvananda/netlink"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	// EncryptionModeDisabled captures enum value "Disabled"
+	EncryptionModeDisabled string = "Disabled"
+
+	// EncryptionModeIPsec captures enum value "IPsec"
+	EncryptionModeIPsec string = "IPsec"
+
+	// EncryptionModeWireguard captures enum value "Wireguard"
+	EncryptionModeWireguard string = "Wireguard"
+)
+
+type void struct{}
+
+var (
+	voidType    void
+	countErrors int
+)
+
+var encryptStatusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Display current state of IPSec configurations",
```

Comment:
This command actually informs about whether WireGuard, or IPSec are enabled or none.
```suggestion
	Short: "Display current state of encryption configurations",
```